src/styles/pages/_live.scss

```diff
@@ -0,0 +1,137 @@
+@import '../settings/colors';
+@import '../tools/breakpoints';
+
+.w-live-landing-page {
```

Comment:
type sizes are a bit off and rather than redo the typography for the site, I'd suggest the easiest path is to do:

```scss
.w-live-landing-page {
  p {
    font-size: 14px;
    line-height: 1.5;
    @include bp(sm) {
      font-size: 16px;
    }
  }
}
```